ds3-sdk/src/test/java/com/spectralogic/ds3client/Ds3ClientBuilder_Test.java

```diff
@@ -31,6 +32,27 @@ public void createBasicClient() throws Exception {
     }
 
     @Test
+    public void createBasicClientFromENV() throws Exception {
+        try {
+            final Ds3ClientBuilder builder = Ds3ClientBuilder.fromEnv();
```

Comment:
It should pass if they aren't set.  If they are set, it verifies nothing is broken in fromEnv() between retrieval of the variables and the return.  Minor and incomplete, I know.